source

```diff
@@ -66141,8 +66142,38 @@ dictionary <dfn>ElementDefinitionOptions</dfn> {
    <span>connected</span>.</dd>
   </dl>
 
+  <p>New instances of <code>CustomElementRegistry</code> can be defined and attached to a
+  <span>ShadowRoot</span> through the registry option from <code
+  data-x="dom-Element-attachShadow">attachShadow()</code>. Those instances are independent of the
+  instance available at <code data-x="dom-window-customElements">window.customElements</code>, as
+  the latter is attached to the
+  <span data-x="concept-document-window">associated <code>Document</code></span>.</p>
+
+  <dl class="domintro">
+    <dt><var>registry</var> = new <code data-x="dom-CustomElementRegistry">CustomElementRegistry</code>()</dt>
+
+    <dd>Defines a new <span data-x="dom-CustomElementRegistry">Registry</span> that can map an
+    independent set of <span data-x="dom-window-customElements">custom elements</span>, which is
+    not associated yet to any <span>ShadowRoot</span>.</dd>
+
+    <dt><var>registry</var> . <code data-x="dom-CustomElementRegistry-define">define</code>(<var>name</var>,
+      <var>constructor</var>)</dt>
+
+    <dd>Defines a new <span>custom element</span> into to given <code data-x="">registry</code>,
```

Comment:
s/into to/within the/

Reply:
Thanks! this is [now fixed](https://github.com/leobalter/html/pull/1/files#diff-41cf6794ba4200b839c53531555f0f3998df4cbb01a4d5cb0b94e3ca5e23947dR66485)!